CoreFoundation/Locale.subproj/CFCalendar.c

```diff
@@ -980,8 +985,13 @@ Boolean _CFCalendarGetComponentDifferenceV(CFCalendarRef calendar, CFAbsoluteTim
             divide = alwaysDivide;
         }
         }
+        if (count < 1) {
```

Comment:
Should this be moved further down (right after the `--`) so we don't go through the loop an additional time and do work just to discover we can't get the difference?

Reply:
It should probably go at the top of the while loop in case `count` starts at `0`. I will fix that

Reply:
Ah, yes — good catch